.github/workflows/hugo.yaml

```diff
@@ -10,11 +10,8 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
 permissions:
   contents: read
```

Comment:
I think these are the default, so `permissions: {}` also works (https://github.com/sigstore/model-transparency/blob/bca8a012a6c29f0f9088f1b46f7150a597fe4f77/.github/workflows/release.yml#L22 works for sure as we have run it before on release)

Reply:
Makes sense. I just left it because it had been there previously and I was trying to avoid breaking anything :-)